pkg/wallet/wallet.go

```diff
@@ -168,9 +168,8 @@ func (w *Wallet) JSON() ([]byte, error) {
 	return json.MarshalIndent(w, " ", "	")
 }
 
-// Close closes the internal rw if its an io.ReadCloser.
-func (w *Wallet) Close() {
-}
+// Close performs nothing and is left for backwards compatibility.
```

Comment:
Mark as deprecated?

Reply:
Fixed.

Reply:
Can't we simply remove this method?

Reply:
Deprecate now, remove later. Not that it's very critical in this particular case (who calls it? removing a line is easy), but we better do it this way where we can.